utils/scarb-test-support/src/expect_dir.rs

```diff
@@ -61,12 +98,50 @@ impl ExpectDir {
             );
         };
 
+        let actual_files = if self.strict {
+            if actual_files.len() != expected_files.len() {
+                panic_on_counts()
+            };
+            actual_files
+        } else {
+            let strip_path = |path: PathBuf, root: &Path| {
+                let path = fsx::canonicalize(&path).unwrap();
+                path.strip_prefix(root).unwrap().to_path_buf()
+            };
+
+            let actual_paths = expected_files
```

Comment:
shouldn't it be `expected_paths`??
```suggestion
            let expected_paths = expected_files
```

Reply:
Oh, yes! Thanks!